### ISSUE TITLE
SUTime extracts dashed numbers as timex duration

### ISSUE THREAD

**Issue by garfieldnate** [stanfordnlp/CoreNLP#145]
Using the demo code from the [website](http://nlp.stanford.edu/software/sutime.html) (except I don't specify the document date), and the input "Call 090-1234-5678", I get this output:

    1234-5678 [from char offset 5 to 18] --> (1234-XX-XX,5678-XX-XX,PT38955312H)

The Timex attributes are:

    {tid=t3, value=PT38955312H, type=DURATION, beginPoint=t1, endPoint=t2}

This happens to be a Japanese phone number, so I understand that it doesn't work right out-of-the-box. However, for extracting `\d\d\d\d-\d\d\d\d` I would expect the resulting timex to be a range, from year 1234 to year 5678, not a duration consisting of the number of hours in that length of time. The referenced [Timex](http://www.timeml.org/tempeval2/tempeval2-trial/guidelines/timex3guidelines-072009.pdf) spec calls this an "anchored duration", and shows it being broken into multiple annotations (page 13), which would be easier to handle.

Even if this does turn out to be incorrectable or unfixable, can you help me find the source of it so I can delete it? I see plenty of duration rules in english.sutime, but they also seem to require other words to match ( like "the" and "year").

**Comment by garfieldnate** [comment]
Here's a partial workaround (where `process(temporal)` is some utility method for handling extracted times):

	SUTime.Temporal temporal = cm.get(
			TimeExpression.Annotation.class).getTemporal();
	if(temporal.getTimexType().equals(TimexType.DURATION)){
		Range range = temporal.getRange(SUTime.RANGE_FLAGS_PAD_AUTO);
                // for anchored durations, process only the anchors
		if(range.begin() != null){
			process(range.begin());
			process(range.end());
		}else
			process(temporal);
	}else{
		process(temporal);
	}

Basically if it's a DURATION type then I check if it is anchored and process the anchors if it is.

This workaround is partial because I don't like the matching in the first place (and need to find and delete the rule), and because I still wonder if there's another way to get a more expected stream of Timex tags. The biggest problem is the (known) lack of documentation.

**Comment by garfieldnate** [comment]
Never mind the "can you help me find the rules part." My ad hoc filtering (removing huge durations) works fine for now and I'm sure you have better things to do.
